en/admin/config.rst

```diff
@@ -1959,6 +1963,23 @@ The following are other parameters. The type and value range for each parameter
         java_stored_procedure_port=4334
         .....
 
+**java_stored_procedure_jvm_options**
+
+    **java_stored_procedure_jvm_options** is a parameter to configure Java Virtual Machine (JVM) and Java options on which Java stored procedures are executed. Each option string should be separated by spaces. For JVM options, there are three types of options; standard, non-standard and advanced options. non-standard and advanced options are not guaranteed to be supported on all VM implementations. If the parameter value configured in [@<database>], it overwrites the value specified in [common]. ::
```

Comment:
What is the default option? Even if it depends on the JVM, I think it would be better to write it down.

Reply:
The default option is an empty string. I'll add about it.